README.md

```diff
@@ -1 +1,39 @@
-# Hello-world
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> Description the project.
```

Comment:
## OPTIONAL
- [ ] Kindly add a description of the project, this will make your readme more professional 👍🏿